### DISCUSSION TITLE
Enabling ctrl+z to suspend the application back to Bash?

### DISCUSSION THREAD

**Discussion by kkinder** [Textualize/textual#3703] [Q&A] [answered]
My Textual app, [Mehditor](https://github.com/kkinder/mehditor), seems to have an inconvenience. There's no good way to suspend the app to the background and resume it later.

Since ctrl+z seems to be overridden, I tried creating a custom action that suspends the process:

    os.kill(os.getpid(), signal.SIGSTOP)

Unfortunately, that doesn't quite work, for two reasons.

1. It doesn't seem like my ctrl+z binding is working
2. If I manually run that code another way, it doesn't clear the screen or restore the cursor; it's just stopped suddenly with the editor still drawn.

Is there a way I can honor ctrl+z like most other terminal apps would? Cheers!

**Comment by 1dancook** [comment, marked as answer]
It's a WIP see #1582 and the linked PR.
I'd love it as well if it was working. That's a pretty crucial thing to have working in a terminal (especially if you are limited to only one window/interface).

**Reply by kkinder** [reply]
Yup, that looks right. I'll revisit it once support is added.

**Reply by kkinder** [reply]
Thanks!
